src/sentry/event_manager.py

```diff
@@ -2257,6 +2270,18 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects):
                 EventPerformanceProblem(event, performance_problems_by_hash[problem_hash]).save()
 
 
+@metrics.wraps("performance.performance_issue.should_create_group")
+def should_create_group(client: RedisCluster, grouphash: str) -> bool:
+    times_seen = client.incr(grouphash)
+    if times_seen >= 3:
+        client.delete(grouphash)
+        return True
+    else:
+        metrics.incr("performance.performance_issue.ignored", 1)
```

Comment:
need some guidance on metric naming

Reply:
Looks good to me, I think as long we we're following the general naming pattern for performance issues we're fine here.

Btw, when calling `.incr` you don't need to pass `1`, that'll be passed by default.